#ifndef PROJECT_VARIABLES_H
#define PROJECT_VARIABLES_H
namespace constants
{
  static int ROBOTS_COUNT;  // TODO: default values from paper
  static double ROBOTS_AVOIDANCE_DISTANCE;
  static double OBSTACLES_AVOIDANCE_DISTANCE;
  static double SENSING_DISTANCE;
  static double LOS_CLEARANCE_DISTANCE;
  static double NEIGHBOURHOOD_DISTANCE;
  static double EDGE_DELETION_DISTANCE;
}
#endif  // PROJECT_VARIABLES_H
